packages/jest-config-terra/src/reporters/verbose-reporter/TerraVerboseReporter.js

```diff
@@ -0,0 +1,116 @@
+const VerboseReporter = require('@jest/reporters/build/verbose_reporter').default;
+const stripAnsi = require('strip-ansi');
+const fs = require('fs');
+const path = require('path');
+const endOfLine = require('os').EOL;
+const { Logger } = require('@cerner/terra-cli');
+
+const LOG_CONTEXT = '[Terra-Toolkit:terra-verbose-reporter]';
```

Comment:
The logger context should be changed to use the prefix. See https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js#L8 for an example. The prefix should changed be `[jest-config-terra:TerraVerboseReporter]`.